pages/shared/reference/materials/materialcreation.md

```diff
@@ -0,0 +1,54 @@
+
+* [Ambient Occlusion](https://help.poliigon.com/en/articles/1712652-texture-maps-explained#h_00c9ca0901) map (Specifies which areas of the texture are "behind", or always shaded. You can compare this to what an [SSBump](https://developer.valvesoftware.com/wiki/$ssbump) map accomplishes but without the interaction with `env_projectedtextures`. For the sake of this article, Ambient Occlusion will be referred to as AO.)
+* [Bump](https://developer.valvesoftware.com/wiki/Bump_map) map *with optional Height map in alpha channel* (Standard bump map that is usually used with standard Source workflow, but it can optionally include a Height map in the alpha channel of the texture for [Parallax Mapping](/p2ce/PBR/parallaxmapping))
+
+For the sake of demonstration, this article will use images of the process of converting `black_wall_metal_005a` to use PBR.
+## Creating the MRAO texture
+### What is "MRAO"?
+A **M**etalness, **R**oughness, **A**mbient **O**cclusion map is a texture that contains the first 3 texture maps listed above in the RGB channels of the texture, respectively (Metalness on the Red channel, Roughness in the Green, AO in the Blue). This texture is used to compress the usually 3 needed maps down into one to free up drive space. While this might not seem like much space, this quickly adds up the more PBR materials your game/mod contains. While most programs do not contain a simple button that does this for you, there are very easy ways to create this texture with basic programs like [GIMP](https://www.gimp.org/), or with specialized programs like [PBR-2-Source](https://koerismo.digital/projects#pbr-2-source). 
+### Method 1: GIMP
+GIMP can combine different layers into the RGBA channels of one texture, which makes creating MRAO maps using GIMP trivial. 
+
+First, go to "Image -> Mode" and make sure "Grayscale" is selected. Even if your images are grayscale, GIMP may still treat them as color images, which prevents this method from working. Then, create a new image with the dimensions of your texture maps. Go to file and click "Import as layers...", select your Metalness, Roughness, and AO maps, and click Import. Your layers should now look similar to this (order is irrelevant):
+
+![Metalness, Roughness, and AO maps in separate layers](https://github.com/Equalizer5118/Wiki/assets/113145248/035e9064-c4f1-4a76-bf3e-8ae822680c18)
+
+After this, go to "Colors -> Components -> Compose...". Put your Metalness map in the Red channel, your Roughness map in the Green, and your AO map in the Blue. The dropdowns should look like this:
+
+![Metalness map in R, Roughness in G, AO in B](https://github.com/Equalizer5118/Wiki/assets/113145248/54573338-4810-4dbe-b3f0-ca74541c9aa8)
```

Comment:
These paths need to be updated, I don't see the assets included in this PR.

Reply:
I ported the images into their own folder in [/assets/PBR_images](https://github.com/Equalizer5118/Wiki/tree/patch-1/assets/PBR_images), and added them into the article in what I think should be fine. I more than likely did something wrong, so I wouldn't be surprised if its broken